app_dart/lib/src/model/luci/push_message.dart

```diff
@@ -148,7 +152,7 @@ class BuildPushMessage extends JsonBody {
 
   @override
   Map<String, dynamic> toJson() => _$BuildPushMessageToJson(this);
-}
+} // end BuildPushMessage
```

Comment:
Remove?

Reply:
Ah yeah, having all these classes was getting crazy and was hard to find things. I will remove this.